src/views/pages/models.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ModelStore from '../../store/model-store.js';
+import CreateModelForm from '../components/create-model-form.jsx';
+import { getAllModels } from '../../lib/api-client'
+export default class Models extends ListenerComponent {
+
+    constructor() {
+        super();
+        getAllModels
```

Comment:
There was me trying to fix a weird bug where if you do ctrl +r the page goes whacky b/c it doesn't have any models, I guess. It didn't work so i'll get rid of it.